tests/mock_network/churn.rs

```diff
@@ -20,65 +20,105 @@ use routing::{
 };
 use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet};
 
-/// Randomly removes some nodes, but <1/3 from each section and never node 0.
-/// Never trigger merge: never remove enough nodes to drop to `elder_size`.
-/// max_per_pfx: limits dropping to the specified count per pfx. It would also
-/// skip prefixes randomly allowing sections to split if this is executed in the same
-/// iteration as `add_nodes_and_poll`.
+/// Randomly removes some nodes.
+///
+/// Limits the number of nodes simultaneously dropped from a section such that the section still
+/// remains functional (capable of reaching consensus), also accounting for the fact that any
+/// dropped node might trigger relocation of other nodes. This limit is currently possibly too
+/// conservative and might change in the future, but it still allows removing at least one node per
+/// section.
+///
+/// If `max_per_pfx` is set, never drops more than that number of nodes per section.
```

Comment:
`max_per_pfx` sounds we shall `never drops the number of nodes per section below that number` ?

Reply:
It's "max drops per prefix". I'll rename the var to make it more obvious.